applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.h

```diff
@@ -64,9 +64,16 @@ double GetRayleighBeta(
     const ProcessInfo& rCurrentProcessInfo);
 
 /**
- * @brief Method to claculate the rayleigh damping-matrix
+ * @brief Method to returns the density to be consider for the mass-matrix computation
+ * @param rElement The Element for which the mass-matrix should be computed
+ * @return The density after apply the mass factor to the element
+ */
+double GetDensity(const Element& rElement);
```

Comment:
Is this really only used in the computation of the MassMatrix?

=> the the method-name should be `GetDensityForMassMatrixComputation` or sth similar

Reply:
Ok